Header                                  header
infrastructure_camera_tracker/Track[]   tracks